app/src/main/java/com/asfoundation/wallet/advertise/WalletPoAService.java

```diff
@@ -85,15 +84,16 @@
         // set the chain id received from the application. If not received, it is set as the main
         String packageName = intent.getStringExtra(PARAM_APP_PACKAGE_NAME);
         requirementsDisposable =
-            proofOfAttentionService.isWalletReady(intent.getIntExtra(PARAM_NETWORK_ID, 1))
+            proofOfAttentionService.isWalletReady(intent.getIntExtra(PARAM_NETWORK_ID, -1))
                 // network chain id
                 .doOnSuccess(requirementsStatus -> proofOfAttentionService.setChainId(packageName,
-                    intent.getIntExtra(PARAM_NETWORK_ID, 1)))
+                    intent.getIntExtra(PARAM_NETWORK_ID, -1)))
```

Comment:
The idea is to have the main network as the default network value when there isn't a value available in the intent extras. Since we no longer will support multiple network at the same time, maybe define the value of the network set as default.

Reply:
This was discussed and in theses cases the user will see a notification (in case he is in one of the 2 networks but on the wrong one) and log the error (when the chain id is not recognisable)